mithril-stm/src/error.rs

```diff
@@ -25,6 +25,10 @@ pub enum MultiSignatureError {
     /// Incorrect proof of possession
     #[error("Key with invalid PoP")]
     KeyInvalid(Box<VerificationKeyPoP>),
+
+    /// At least one signature in the batch is invalid
+    #[error("One signature in the batch is invalid")]
+    BatchInvalid,
```

Comment:
It looks like this error is never used.
Also would it be possible to add the index of the signature that failed?

Reply:
I've included the error, but unfortunately we cannot add the index of the signature that failed, as we don't know. We simply know that the batch is invalid. If we get this error, we need to individually verify each signature.